### DISCUSSION TITLE
[Question][Hw4] .cov file not generated

### DISCUSSION THREAD

**Discussion by 0xXcrypted** [prosyslab-classroom/cs348-information-security#576] [Q&A]
I followed the instruction from the Readme.md
`make`
`cd test; make`
`../fuzzer example1 string_seed output_dir`
but this gives me error with
`Fatal error: exception Sys_error("~~/test/example1.cov: No such file or directory")`
I think this error is strange because make command on root and test directory should make .cov file.

To be clear, I copied instrument.ml and modified to ignore PHI case.

**Comment by hieudz** [comment]
Did you paste your project 3 code into the src folder?

**Comment by 0xXcrypted** [comment]
I ran the command from HW3
`./sanitizer test/example1.ll` 
`clang -o example1 test/example1.instrumented.ll test/runtime.c`
and this gives me exampl1.cov file on the root directory.
Is it okay for me to copy paste this file into the test directory and run HW4 commands?

**Comment by 0xXcrypted** [comment]
I thought using test and Coverage.read in a line would be okay but it wasn't
closing : solved issue
